cadf/init.sls

```diff
@@ -4,6 +4,7 @@
 include:
 - cadf.dispatcher
 - cadf.listener
+- cadf.logrotate
```

Comment:
not this way: Define logrotate like: https://github.com/salt-formulas/salt-formula-linux/blob/master/linux/meta/logrotate.yml

So in ```cadf/files/logrotate```
salt-formula-cadf/cadf/meta/logrotate.yml``` store the configs you have in